docs/Reference/CLI/CLI-Syntax.md

```diff
@@ -328,9 +328,15 @@ Show the help message and exit.
     initial-state: "/home/me/genesis.ssz"
     ```
 
-Path or URL to the network genesis file.
+Path or URL to an SSZ encoded state file. The state file can be used to specify the genesis state,
+or a recent state from which to sync.
 
-This option does not need to be specified if the initial state is provided by the network specified
+!!! note
+
+    If overriding the initial state in a custom network. You will need to supply the initial state
+    file at each restart.
```

Comment:
@mbaxter Can you confirm if this applies to all network or only custom networks?

Reply:
This should only apply to custom networks.  

The issue would show up if you're using a known network definition ("mainnet", "medalla", etc) which has a default initial state (the genesis state).  If you override the initial / genesis state, you need to keep overriding the initial state.  Otherwise, if you start up without the initial state configured, the node will load the default initial state and try to validate the existing chain data against the default initial state (which will fail).